### PR TITLE
Reorganize content for load testing 

### PR DESCRIPTION
**Pull request by aliciakrebs**
Reorganize content for load testing

### Description
Reorganize content for load testing 

### Motivation and Context
Improve flow of the Load Testing with API Testing content.
[TCD-2683](https://saucedev.atlassian.net/browse/TCD-2683)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

[TCD-2683]: https://saucedev.atlassian.net/browse/TCD-2683?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lysannep — APPROVED** [review]

File: docs/api-testing/load-testing.md [hunk 1]
```diff
@@ -75,14 +71,22 @@ Variables and snippets in the Vault (Company/Project) are used by default, witho
 
 You can set up multiple profiles for each test, from less aggressive to very aggressive, or with varying durations. The only limitations are those inherited from your infrastructure.
 
-Once you have created and saved a load test, the configuration will be saved for future use. To access the list of previously saved configurations for a specific test, click the load test icon in the test list.
+<img src={useBaseUrl('/img/api-testing/create-load-test.png')} alt="The Create Load test page"/>
+
+5. Click **Save and Run**. 
+
+:::note
+Load testing only works with published tests. If you change a published test, and then run a load test without re-publishing it, the load test will be run on the published version, not the updated one.
```
Comment (LoryApiFortress) [inline docs/api-testing/load-testing.md:79]: @aliciakrebs I think the note should be after point 2. Users can reach the load test page also from within the composer this means they can change the test and select the load test link without publishing it, so it is better they are aware of it at the start, not when they've already done all the things

**Review by LoryApiFortress — COMMENTED** [review]
@aliciakrebs apart for the suggestion to move a note, looks good

**Review by cb-sl — APPROVED** [review]
Looks good, thank you. (Loredana's feedback makes sense, happy for that to be included and pusblished)

File: docs/api-testing/load-testing.md (hunk 1 above)
Comment (aliciakrebs) [reply docs/api-testing/load-testing.md]: I moved the note to Step 2.

**Pull request merged**